src/include/daos_srv/ras.h

```diff
@@ -33,29 +33,26 @@
  * In order to minimize conflicts between patches, please:
  *   * Don't change the first and last entries in the list
  *   * Don't arbitrarily reorder entries
- *   * Do limit lines to 73 columns, wrapping as necessary
+ *   * Do limit lines to 99 columns, wrapping as necessary
  */
-#define RAS_EVENT_LIST							\
-	X(RAS_UNKNOWN_EVENT,		"unknown_ras_event")		\
-	X(RAS_ENGINE_FORMAT_REQUIRED,	"engine_format_required")	\
-	X(RAS_ENGINE_DIED,		"engine_died")			\
-	X(RAS_ENGINE_ASSERTED,		"engine_asserted")		\
-	X(RAS_ENGINE_CLOCK_DRIFT,	"engine_clock_drift")		\
-	X(RAS_POOL_CORRUPTION_DETECTED,	"corruption_detected")		\
-	X(RAS_POOL_REBUILD_START,	"pool_rebuild_started")		\
-	X(RAS_POOL_REBUILD_END,		"pool_rebuild_finished")	\
-	X(RAS_POOL_REBUILD_FAILED,	"pool_rebuild_failed")		\
-	X(RAS_POOL_REPS_UPDATE,		"pool_replicas_updated")	\
-	X(RAS_POOL_DF_INCOMPAT,						\
-	  "pool_durable_format_incompatible")				\
-	X(RAS_POOL_DEFER_DESTROY,	"pool_destroy_deferred")	\
-	X(RAS_CONT_DF_INCOMPAT,						\
-	  "container_durable_format_incompatible")			\
-	X(RAS_RDB_DF_INCOMPAT,						\
-	  "rdb_durable_format_incompatible")				\
-	X(RAS_SWIM_RANK_ALIVE,		"swim_rank_alive")		\
-	X(RAS_SWIM_RANK_DEAD,		"swim_rank_dead")		\
-	X(RAS_SYSTEM_START_FAILED,	"system_start_failed")		\
+#define RAS_EVENT_LIST									\
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
can we please ignore, only line formatting was changed